### PR TITLE
Left align supporters in footer

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/pages/BrandedPage.scss [hunk 1]
```diff
@@ -28,7 +28,18 @@
         }
 
         .supporters {
-            text-align: center;
+            text-align: left;
+            display: inline;
+            padding: 0;
```
Comment (danni) [inline src/pages/BrandedPage.scss:33]: `white-space: nowrap`

File: src/pages/BrandedPage.scss [hunk 2]
```diff
@@ -28,7 +28,18 @@
...
+        }
+
+        .supporter {
+            display: inline-block;
+            list-style-type: none;
+        }
+
+        .supporter + .supporter::before {
+            content: "\00a0|\00a0"
```
Comment (danni) [inline src/pages/BrandedPage.scss:42]: Nice

File: src/pages/BrandedPage.scss (hunk 1 above)
Comment (DanielHeath) [inline src/pages/BrandedPage.scss:33]: `inline-block` breaks first

**Comment by danni** [comment]
++

**Pull request merged**